docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3.4'
+services:
+ sourcecred:
+  container_name: sourcecred
+  image: sourcecred:latest
+  build:
+   context: .
+   dockerfile: docker/Dockerfile
+   cache_from:
+   - sourcecred:latest
+   - ubuntu:18.04
+  environment:
+  - SOURCECRED_GITHUB_TOKEN
+  hostname: sourcecred
+  #network_mode: host
+  restart: unless-stopped
```

Comment:
I wouldn't restart on failures right now. It could lead to some considerable API spamming because of the load command.